registration.html

```diff
@@ -0,0 +1,107 @@
+            <div class="col-sm-3">
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-sm-3">
+            </div>
+            <div class="col-sm-3">
+                <label for="agecheck">Your age in years</label>
+                <input type="number" size=65 name="yearuser" class="form-control" id="agecheck"
+                    placeholder="type number" required>
+            </div>
+            <div class="col-sm-3">
+            </div>
+            <div class="col-sm-3">
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-sm-3">
+            </div>
+            <div class="col-sm-3">
+                <label for="city">City</label>
+                <input type="text" size=40 name="city" class="form-control" id="city" placeholder="munich" required>
+            </div>
+            <div class="col-sm-3 autocomplete">
+                <label for="validationCustom05">Country</label>
+                <input class="form-control" id="country" type="text" name="country" placeholder="Country" required>
+            </div>
+            <div class="col-sm-3">
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-sm-3">
+            </div>
+            <div class="col-sm-3">
+                <label for="email">Email address:</label>
+                <input type="email" size=65 name="email" class="form-control" id="email"
+                    placeholder="type email address" required>
+            </div>
+            <div class="col-sm-3">
+                <label for="emailconfirm">Confirm email address:</label>
+                <input type="email" size=65 name="confirmemail" class="form-control" id="emailconfirm"
+                    value="retype email address" required>
+            </div>
+            <div class="col-sm-3">
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-sm-3">
+            </div>
+            <div class="col-sm-3">
+                <label for="password">Password:</label>
+                <input type="password" size=65 name="passworduser" class="form-control" id="password"
+                    placeholder="type password" required>
+                <div class="form-check">
+                    <input name="TAC" class="form-check-input" type="checkbox" id="gridCheck">
+                    <label class="form-check-label" for="gridCheck">I agree with <a href="">Terms &
+                            Conditions</a></label>
+                    </label>
+                </div>
+                <div class="col-sm-3">
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-sm-3">
+                </div>
+                <div class="col-sm-3">
+                </div>
+                <p></p>
+                <div class="col-sm-3">
+                    <button class="btn btn-danger" type="submit">Create an account</button></div>
```

Comment:
where is btn-danger style is coming from? I can't find it...

Reply:
From bootstrap pre-set styles in the <link ref=""> on line 8